lib/src/Pos/Block/Logic/Util.hs

```diff
@@ -150,7 +151,7 @@ calcOverallChainQuality =
     getCurrentSlotFlat >>= \case
         Nothing -> pure Nothing
         Just curFlatSlot ->
-            calcOverallChainQualityDo curFlatSlot <$> DB.getTipHeader @ssc
+            calcOverallChainQualityDo curFlatSlot <$> DB.getTipHeader @SscGodTossing
```

Comment:
I hope this type application won't be needed after CSL-1279 is fully finished.

Reply:
Yep.